proposals/incubation/kubevirt.adoc

```diff
@@ -0,0 +1,164 @@
+
+The project uses two channels on the CNCF Kubernetes Slack server. #virtualization is used to handle general use topics. The channel hosts over 1000 users. #kubevirt-dev is used for developer oriented topics. The channel hosts 350 users. The channels typically receive 0-5 new threads per day.
+
+Important announcements are relayed via mailing list, website blog and Twitter.
+
+KubeVirt advertises communications channels via https://kubevirt.io/community as well as project [README](https://github.com/kubevirt/kubevirt/blob/master/README.md).
+
+### End Users
+
+Project end users are maintained in the [Adopters](https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md) files
+
+Significant users include:
+* NVIDIA
+* SUSE
+* Kubermatic
+* H3C
+* CoreWeave
+* Civo
+* CloudFlare
+* Ateme
+* Cloudbase Solutions
+
+### Integrations
+
+* [oVirt](https://www.ovirt.org) ... Virtual machines can be run in a container using the external provider feature.
+* [Gardener](https://gardener.cloud/blog/2020-10/00/) ...
+* [Kubermatic Virtualization](https://www.kubermatic.com/products/kubevirt/)
+* [SUSE/Rancher Harvester](https://github.com/rancher/harvester/blob/766abd06561b059c1af623aacc4e505db471ceee/deploy/charts/harvester/README.md)
+* [Google Anthos](https://youtu.be/RE0A3kHT3LA?t=126)
+
+### CNCF Sponsored Security Audit
+
+## Product information
+
+### Feature design proposals
+Design proposals to allow community members to gain feedback on their designs from the repo approvers before the community member commits to executing on the design. By going through the design process, developers gain a have a high level of confidence that their designs are viable and will be accepted.
+
+* [design-proposals](https://github.com/kubevirt/community/tree/master/design-proposals)
+
+### Release cadence
+
+Kubevirt has an established and documented release process and cadence
+
+* [Release process](https://github.com/kubevirt/kubevirt/blob/main/docs/release.md)
+* [Release cadence](https://github.com/kubevirt/kubevirt/blob/main/docs/release.md#cadence-and-timeline)
+
+### Delivered features
+
+* [DONE] GA v1 API for core KubeVirt APIs
+ * API v1 features need to rely on GA’ed Kubernetes entities, fully fledged (incl e.g. explain, validation)
+ * An OpenAPI definition as the only source of truth for KubeVirt’s API
+ * https://github.com/kubevirt/kubevirt/pull/3349
+* [DONE] Zero downtime live updates
+* [DONE] Stabilize bridge network binding
+* [DONE] Disk hotplug
+* [DONE] IPv6 support
+* [DONE] Device passthrough support
+* [DONE] Numa topology support
+* [DONE] Memory metrics gathering support
+* [DONE] Affinity / Anti-Affinity rules
+* [DONE] Live-Migration support
+* [DONE] Offline disk snapshots
+* [DONE] SRIOV support
+* [DONE] Dynamic SSH Key Injection
+* [DONE] Multus support for multiple network interfaces attached to Virtual Machines
+* [DONE] Dedicated prow deployment for CI functional tests and automation
+
+### Future Roadmap
+
```

Comment:
Here we should add a note that the future roadmap is being mapped out at https://github.com/kubevirt/kubevirt/milestones